tools/setup

```diff
@@ -37,7 +37,7 @@ status "3. Update npm to latest version"
 nvm install-latest-npm  || fail 'Failed to update npm to latest version.'
 
 status "4. Install dependencies"
-(cd $ROOT && npm ci) || fail 'Failed to install dependencies.' 
```

Comment:
Travis has added support for Windows recently and we've had brief discussion on exploring that at some point instead of AppVeyor. If we move to using the `setup` script, then we'd need to do add'l work to try that -- creating a `.bat` version, and some conditional logic on which script to run, etc.

Will our AppVeyor build still work after this patch? Is it further deviant, or at risk of becoming deviant, from our non-Windows build node/npm versions after this patch?

Reply:
Appveyor will still work after this patch. 
I have another branch in progress #3396 to bring about a Windows build.